src/main/java/net/snowflake/ingest/utils/Utils.java

```diff
@@ -74,6 +74,7 @@ public static void assertNotNull(String name, Object value) throws SFException {
    */
   public static Properties createProperties(Properties inputProp, boolean sslEnabled) {
     Properties properties = new Properties();
+    properties.putAll(inputProp);
```

Comment:
I think we would still want to use the switch case below to add to the properties, it does more things like toLowerCase and log a entry if the key is not useful.

Reply:
Please double check my logic, but i think I kept all of the reformatting we do below and figured it was simpler to just pass along all the props even if we don't use them

Reply:
hmmm, IIUC, you remove a few cases in the switch statement below, so let's say if the input prop is `USER` instead of `user`, we won't able to get it later anymore with our change, right? Also now we will log an entry saying user property is ignored which is not correct. I feel like the old logic is more clear, could we change it back?